how-to-run-locally.md

```diff
@@ -0,0 +1,96 @@
+# How to run altualizr-lite locally
+The following is a guide on how to run aktualizr-lite on your local Linux-based host against the Foundries Factory.
+
+## Work dir setup
+
+`mkdir local-device-001 && cd local-device-001`
+
+### Optionally setup docker-compose and docker for your setup
+#### Docker-compose
+```
+python3 -m venv venv
+python3 -m pip install --upgrade pip
+python3 -m pip install docker-compose
```

Comment:
These python invocations should use the venv right?

Reply:
yep, fixed.